cpu/stm32l1/periph/i2c.c

```diff
@@ -34,17 +34,14 @@
 #define ENABLE_DEBUG    (0)
 #include "debug.h"
 
-/* guard file in case no I2C device is defined */
-#if I2C_NUMOF
-
```

Comment:
Hm? This is what the gaurd intents: `guard file in case no I2C device is defined`